files/workflows/ZWEWRF04.xml

```diff
@@ -0,0 +1,177 @@
+        <template>
+            <inlineTemplate substitution="true">mkdir -p "${instance-zowe_install_path}"</inlineTemplate>
+            <submitAs maxRc="0">shell-JCL</submitAs>
+            <maxLrecl>1024</maxLrecl>
+            <!-- zosmfOutput -->
+        </template>
+    </step>
+    <step name="mount_zfs" optional="false">
+        <title>Mount ZFS</title>
+        <description>Mounts the ZFS data set to the created directory</description>
+        <prereqStep name="define_variables"/>
+        <prereqStep name="allocate_zfs"/>
+        <!--variableValues-->
+        <!-- condition -->
+        <instructions substitution="false">Run this step to mount the ZFS data set</instructions>
+        <weight>1</weight>
+        <skills>None</skills>
+        <autoEnable>true</autoEnable>
+        <canMarkAsFailed>false</canMarkAsFailed>
+        <rest>
+            <httpMethod>PUT</httpMethod>
+            <uriPath substitution="true">/zosmf/restfiles/mfs/${instance-zowe_dsn_prefix}.ZOWEZFS</uriPath>
+            <requestBody substitution="true">{
+"action":"mount",
+"mount-point":"${instance-zowe_install_path}",
+"fs-type":"ZFS",
+"mode":"rdwr"}</requestBody>
+            <expectedStatusCode>204</expectedStatusCode>
+            <!-- REST mappings -->
+        </rest>
+    </step>
+    <step name="set_owner" optional="false">
+        <title>Set Mountpoint Owner</title>
+        <description>This step will set the person running the step as the owner of the mountpoint.</description>
+        <prereqStep name="mount_zfs"/>
+        <!--variableValues-->
+        <!-- condition -->
+        <instructions substitution="false">Run this step to set you as the owner of the mountpoint.</instructions>
+        <weight>1</weight>
+        <skills>z/OS UNIX Administrator</skills>
+        <autoEnable>true</autoEnable>
+        <canMarkAsFailed>false</canMarkAsFailed>
+        <template>
+            <inlineTemplate substitution="true">echo "chown ${_step-stepOwnerUpper} ${instance-zowe_install_path}" | su</inlineTemplate>
+            <submitAs maxRc="0">shell-JCL</submitAs>
+            <maxLrecl>1024</maxLrecl>
+            <!-- zosmfOutput -->
+        </template>
+    </step>
+    <step name="run_install_script" optional="false">
+        <title>Run install script</title>
+        <description>This step will run the Zowe convenience build install script.</description>
+        <prereqStep name="define_variables"/>
+        <!--variableValues-->
+        <!-- condition -->
+        <instructions substitution="true">Run this step to run the Zowe install script.</instructions>
+        <weight>1</weight>
+        <skills>None</skills>
+        <autoEnable>true</autoEnable>
+        <canMarkAsFailed>false</canMarkAsFailed>
+        <template>
+            <inlineTemplate substitution="true">set -x
+cd "${instance-zowe_build}/install"
+sh ./zowe-install.sh -i ${instance-zowe_install_path} -h ${instance-zowe_dsn_prefix}</inlineTemplate>
```

Comment:
sh TMPDIR=${"${our_tmpdir}":-/tmp}; .zowe-install.sh ...

Reply:
I have made just a simple TMPDIR assign, because inside the `zowe-install.sh` script there already is:

```sh
export TEMP_DIR=${TMPDIR:-/tmp}/zowe_"`date +%Y-%m-%d`"
```